### ISSUE TITLE
[BUG] examples not loading

### ISSUE THREAD

**Issue by prasun3** [pyscript/pyscript#277]
**Describe the bug**

I am trying to see the examples but I am unable to do so. I downloaded the repo and opened the examples in Chrome, but the examples didn't work. For example this is what antigravity.html rendered.
```
- paths: - ./antigravity.py Based on xkcd: antigravity https://xkcd.com/353/. import antigravity antigravity.fly()
```
I think this is because the examples refer to `../build/pyscript.{css,js}` which is not present in the repo. I created the folder and downloaded `pyscript.{css,js}' and reloaded the page but saw the following error in the browser.
```
Access to fetch at 'file:///C:/path/to/pyscript/pyscript-main/pyscriptjs/examples/antigravity.py' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https.
```

**To Reproduce**
Download repo and open in Chrome

**Expected behavior**
See examples in action.

**Screenshots**
None

**Additional context**
None

**Comment by marcbernot** [comment]
See explanation and workaround there #257.

**Comment by antocuni** [comment]
Duplicate of #257
